js/components/ImportIndicatorsPopover.js

```diff
@@ -1,23 +1,97 @@
 import React, { useState, useContext, useEffect } from 'react';
+import ReactDOM from 'react-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import Select from 'react-select';
-import { BootstrapPopoverButton } from './helpPopover';
 import api from '../apiv2';
 
 export const ImportIndicatorsContext = React.createContext();
 
-export class ImportIndicatorsButton extends BootstrapPopoverButton {
-    // Overriding variables in the BootstrapPopoverButton
+// **********************************************
+// ***** Import Indicators Button Component *****
+// **********************************************
+export class ImportIndicatorsButton extends React.Component {
     popoverName = "importIndicators";
     popoverTitle = gettext("Import indicators");
 
     constructor(props) {
         super(props);
+        this.state = {
+            inactiveTimer: null,
+            tierLevelsUsed: [],
+            storedView: {}, // Store the current popover view, valid, and/or invalid rows to reopen where left off if closed
+            setStoredView: this.setStoredView,
+            storedTierLevelsRows: [], // Store the popover's desired tier level row counts to reopen where left off if closed
+            setStoredTierLevelsRows: this.setStoredTierLevelsRows,
```

Comment:
setStoredTierLevelsRows probably doesn't need to be in state if it's properly bound to the component